### DISCUSSION TITLE
A simple question about loading pretrained model

### DISCUSSION THREAD

**Discussion by ThomaswellY** [open-mmlab/mmpose#2313] [General]
I wonder what's the differences between the two methods of paras configuration
1. load_from = '/path/pretrain_model.pth'
   resume = True
2. model=dict(
……
init_cfg=dict(type='Pretrained',prefix='backbone.',
checkpoint='/path/pretrain_model.pth')
)
what's more, how to fine tune specific layers( like  full fine-tuning and linear probing )through these two methods

**Comment by Tau-J** [comment]
`load_from` will load all weights into the model, while `prefix='backbone.'` will only load the pretrained weights into the backbone.

**Reply by ThomaswellY** [reply]
Thanks alot, so the para configurations i listed were all used to train the model(whole or only backbone), then how can i execute fine tuning? I haven't seen any operating guide in doc in mmlab project.

**Reply by Tau-J** [reply]
emmm.. tbh, currently `mmengine` did not support freeze layers by config, [this PR](https://github.com/open-mmlab/mmengine/pull/767) is under review. I'm afraid that you have to freeze modules yourself in the model.
